cores/epoxy/Client.h

```diff
@@ -37,22 +35,23 @@ class Client: public Stream {
         virtual size_t write(const uint8_t *buf, size_t size) override = 0;
         virtual int available() override = 0;
         virtual int read() override = 0;
-        /* virtual int read(uint8_t *buf, size_t size) override = 0; */
+        virtual int read(uint8_t *buf, size_t size) = 0;
```

Comment:
I see, this method is added in Client.h and does not exist in Stream.h. Looks good.